src/BLSSignatureChecker.sol

```diff
@@ -28,12 +28,22 @@ contract BLSSignatureChecker is IBLSSignatureChecker {
     IStakeRegistry public immutable stakeRegistry;
     IBLSPubkeyRegistry public immutable blsPubkeyRegistry;
     IDelegationManager public immutable delegation;
+    IServiceManager public immutable serviceManager;
+    /// @notice If true, check that the signature timestamp is within the delegation withdrawalDelayBlocks window.
+    bool public isTimestampChecked;
```

Comment:
prefer something like `isStakeUpdateChecked`

Reply:
perhaps `timelyUpdatesRequired` or something?
maybe `staleStakesForbidden` ?

Reply:
got G's opinion offline, feels like `staleStakesForbidden` is best -- it feels succinct-ish with the most clarity of meaning.
`isTimestampChecked` and `isStakeUpdateChecked` beg the question "check for what / checked how?", and `timelyUpdatesRequired` begs the question "what kind of update?"

Reply:
could be good to workshop /  ask a bit more broadly for ideas tho

Reply:
I was also iffy on my initial naming as its not exactly clear what is being checked. `staleStakesForbidden` sounds like a much better alternative but I'll ask for more input

Reply:
Also forgot as well its not timestamp based anymore, its based on blocks too